### ISSUE TITLE
Make it easier to recover files?

### ISSUE THREAD

**Issue by jayongg** [Smurf-IV/Elucidate#5]
I would _love_ a GUI to pick/select folders/files to recover from the .content files, and recover either a whole drive or files over to a folder.

Something  you might be interested in? :)

**Comment by jayongg** [comment]
Ahh, I think this is part of Phase IV.  Theoretically that can be done in parallel with phase 3 if you had enough people right?

**Comment by Smurf-IV** [comment]
Enough people - yes, **and** I had finished refactoring / redesigning / catching up with .Net 4.7

**Comment by jayongg** [comment]
Are you looking for help these days? One thing I also noticed is that the scheduling only lets you get one schedule, is that by design?  (Let me know if I should open another issue). So eg I can’t set a schedule for sync then another to scrub.

**Comment by Smurf-IV** [comment]
Help Yes - Soon, You'll see when I finally get the next "Stable" image up into Git.
Single Schedule was by design, because when I first used the Third-party dll it had a few troubles (And was slow !) - So please open another issue / request with ideas for that.

**Comment by jayongg** [comment]
Thanks!

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
